app/cpp/shaders/CMakeLists.txt

```diff
@@ -1,4 +1,9 @@
-set(glslc_exe "${CMAKE_ANDROID_NDK}/shader-tools/${CMAKE_ANDROID_NDK_TOOLCHAIN_HOST_TAG}/glslc")
+IF (WIN32)
```

Comment:
according to the [cmake documentation](https://cmake.org/cmake/help/latest/variable/WIN32.html) this flag is set to True when the target system is Windows. As far as I know, the target is the target platform the project is being built for and not the host platform. `CMAKE_HOST_SYSTEM_NAME` should be used in here.